templates/web/login.ftl

```diff
@@ -40,7 +40,9 @@
 <#include "/templates/web/common/js-next-scripts.ftl" />
 <script>
   CrafterCMSNext.render('#root', 'Login', {
+    <#outputformat "HTML">
     passwordRequirementsRegex: '${passwordRequirementsRegex}'
```

Comment:
Did you test and review this well ? Are all the chars of the default regex still rendered identically? Same on studio-context.ftl.